doc/NotaryContentTrust.md

```diff
@@ -73,7 +73,7 @@ Notary initializes the TUF trust collection for each image in the Container Regi
 ##### a. Notary Installation
 Notary client can be installed with the command for `amd64` target platform and avoid installing using `sudo apt-get install notary`
```

Comment:
Notary can be installed with the following wget command for `amd64` target platform.  Avoid installing using `apt-get`.  
(Why should we avoid installing via apt?)

Reply:
ACR notary contact mentioned that the list is not update regularly and we would want a specific version. so this method.

Reply:
I noticed the copy paste. Will remove that